run.sh

```diff
@@ -6,59 +6,75 @@ set -e # exit on errors
 ORG=$1
 PACKAGE=$2
 EXECUTABLE=$3
-if [[ -z "$ORG" || -z "$PACKAGE" || -z "$EXECUTABLE" ]]; then
-    echo No arguments provided, please pass the org, repo and executable name as the first, second and third arguments
-    exit 1
+VERSION=$4
+
+if [[ -z "$ORG" || -z "$PACKAGE" || -z "$EXECUTABLE" || -z "$VERSION" ]]; then
+  echo No arguments provided, please pass the org, repo, executable name and version as the first, second, third and fourth arguments
```

Comment:
I should have changed the wording to something like `first 4 arguments` instead of continuing counting